akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -12,7 +12,7 @@ If you are still using Scala 2.11 then you must upgrade to 2.12 or 2.13
 
 ### akka-camel removed
 
-After being deprecated in 2.5.0, the akka-camel module has been removed in 2.6.
+After being deprecated in `2.5.0`, the akka-camel module has been removed in `2.6`.
```

Comment:
Some have back tiks and some versions do not but it seems most did. Let's pick a convention for everywhere moving forward.

Reply:
It's not code so no code-ticks would be my vote.

Reply:
WFM 👍 changing